src/utils/time.ts

```diff
@@ -9,18 +9,10 @@ export const epochToMoment = (epoch: number) => moment.unix(epoch).utc();
  * Function that takes a primitive type and converts it into a Moment instance
  */
 export const toMoment = (value?: moment.MomentInput): moment.Moment => {
-    if (!value) return moment().utc(); // returns 'now' moment object
+    if (!value || !moment(value).isValid()) return moment().utc(); // returns 'now' moment object
```

Comment:
reason why i delted the below if check for invalid is because it really didn't make sense to have it. If the value is invalid, it wouldn't be able to calculate any of the values below properly, it would return NaN, like if you pass a string thats not in date format. Also, I opted to use the isValid check from moment, as this is plenty enough to handle if it is a valid moment value, and we can return the default as the current now moment object.

I also checked throughout the code base where we are using toMoment, and did a quick sanity check, all looks good.